docs/blog/2019-01-29-themes-update-child-theming-and-component-shadowing/index.md

```diff
@@ -0,0 +1,131 @@
+
+## A Concrete Example
+
+Taking advantage of child theming requires some familiarity with theming itself
+to be used effectively. If you're already experienced with themes and want to
+see how one possible way to upgrade your themes to use this feature, the
+corresponding PR in the [gatsby-theme-examples
+repo](https://github.com/ChristopherBiscardi/gatsby-theme-examples) has [also
+been
+merged](https://github.com/ChristopherBiscardi/gatsby-theme-examples/pull/13).
+
+Let's say you have a theme that contains a blog data model and some logic
+relating to how to render blog posts at specific pages, we'll call it
+`gatsby-theme-blog-core`. This theme renders JSON representations of blog posts
+at the relevant page locations instead of full UI because it's only meant to
+encapsulate the data model. We'll skip going over page creation logic in
+`gatsby-node.js` and instead focus on the following possible `gatsby-config.js`
+for the core theme.
+
+```js
+module.exports = {
+  plugins: [`gatsby-mdx`],
+}
+```
+
+Then we have another theme that handles applying a UI layer using to the core
+data model, `gatsby-theme-blog`. This theme is a child theme of
+`gatsby-theme-blog-core` and shadows the core theme's modules to render
+beautiful posts on each blog using any styling technology we want, like
+[Emotion](https://emotion.sh/) or [Sass](https://sass-lang.com/). Again, since
+this is a post on child theming itself we'll skip looking at how shadowing is
+applied to the parent and focus on one possible child theme `gatsby-config.js`.
+
+```js
+module.exports = {
+  __experimentalThemes: [`gatsby-theme-blog-core`],
+  plugins: [`gatsby-plugin-emotion`],
+}
+```
+
+Now that we've composed the original data model (with page creation logic) with
+the plugins and shadowed components to build the UI, any user can use our theme
+by specifying only the child in their `gatsby-config.js`.
+
+```js
+module.exports = {
+  __experimentalThemes: [`gatsby-theme-blog`],
+}
+```
+
+The end result from a theme's composition view is that we apply the parent, then
+the child, then the user's site. This opens up a whole host of additional
+possiblities as child themes can have multiple parents or compose with each
+other. Data models for blogging and ecommerce can be combined into a usable
+child theme and distributed as a single npm package. Users don't need to know
+your theme is made up of multiple parents.
+
+If we go back to our list of use cases, we can map each of them to a layer in
+this theme stack. The designer uses the child theme to make their own site
+(passing tokens into the theme config), the person proficient in HTML and CSS
+can shadow components from the core theme (using minimal React), and the Gatsby
+expert can craft custom data models for their specific use cases to compose with
+other models. Each link in the chain supports the next allowing us to develop a
+progressive disclosure of complexity for people with different skillsets.
```

Comment:
Is there a simpler way to say "progressive disclosure of complexity" to make it a little easier on the reader?